lib/virtfs/ntfs/attrib_volume_name.rb

```diff
@@ -1,5 +1,3 @@
-require 'util/miq-unicode'
```

Comment:
Not sure, but you might be a `require 'fs/ntfs/utils'` here like is done in the other places.

Reply:
Done.

Reply:
@Fryguy Actually, where is this coming from?

Reply:
@Fryguy It looks to me like this library assumed manageiq-smartstate was a dependency, but it's not marked as such, and that's a different utils.rb file anyway.

Reply:
oh very interesting...I missed that detail

Reply:
The virtfs code was expected to be extracted _from_ manageiq-smartstate.  In fact, I'd expect you to run into duplicated code.  Note that a lot of this code doesn't actually work yet, and was in progress, but has since been abandoned.  It is not used at all by the application.

On that note, I think we should abandon this virtfs stuff for now (like completely abandon, archive the repos)

Reply:
@Fryguy I'm all in favor of archiving this if it's not actually being used.

Reply:
Seeing as these virt* repos have not been active for awhile and just cause noise (needless PRs and Issues) due to gem and cop conflicts, let's archive them for now.  If someone wants to work on them in earnest, we can always un-archive them.  /cc @roliveri